internal/setup/openpgp.go

```diff
@@ -0,0 +1,76 @@
+package setup
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+
+	"golang.org/x/crypto/openpgp/armor"
+	"golang.org/x/crypto/openpgp/clearsign"
+	"golang.org/x/crypto/openpgp/packet"
+)
+
+// DecodeKeys decodes public and private key packets from armored data.
+func DecodeKeys(armoredKey []byte) (pubKeys []*packet.PublicKey, privKeys []*packet.PrivateKey, err error) {
```

Comment:
Ya I need to decode private keys as well, in tests. Rather than creating another function which decodes a single private key, I would rather have this since this is less restrictive.